content/posts/Podcast: A big ol' cloud of compute.md

```diff
@@ -0,0 +1,23 @@
+---
+title: "Podcast: A big ol' cloud of compute"
+date: 2019-03-19T10:17:00
+---
+
+John and Justin made a guest appearance recently on the 'Deloitte: On Cloud' podcast with David Linthicum, talking about cloud native, DevOps, and Kubernetes from the business perspective.
+
+> What I want is just a big ol' cloud of compute and I can just stuff my container in it and have it work. And we kind of have that now with Kubernetes. It's not that there's anything so magical about Kubernetes itself so much as the fact that it's just universally adopted now.<br>
```

Comment:
I feel sorry for the poor transcribing person who had to try and make sense of my disconnected ramblings...